.gitignore

```diff
@@ -23,4 +23,5 @@ hs_err_pid*
 
 # Temp files used for testing
 test/actual.txt
-test/localrun.bat
\ No newline at end of file
+test/localrun.bat
+.DS_Store
```

Comment:
This change should not be in this PR. Check the staged changes before committing to ensure it doesn't contain unintended changes.